pkg/standalone/standalone.go

```diff
@@ -569,6 +571,15 @@ func createComponentsAndConfiguration(wg *sync.WaitGroup, errorChan chan<- error
 	}
 }
 
+func createSlimConfiguration(wg *sync.WaitGroup, errorChan chan<- error, _, _ string, _ string) {
+	defer wg.Done()
+	err := createDefaultConfiguration("", DefaultConfigFilePath())
```

Comment:
add comment here passing empty zipkin host for slim init.

Reply:
Sure